app/facades/search_facade.rb

```diff
@@ -0,0 +1,13 @@
+class SearchFacade
+  def initialize(params)
+    @is_remote = params[:is_remote]
+    @user_id = params[:user_id]
+    @query = params[:query]
+  end
+
+  def build_users
+    users = User.search_for_skills(@query)
+    users = users.remote_users if @is_remote
+    users.map { |user| SearchedUserPoro.new(user) }
```

Comment:
This is clean and I like where it's going, essentially filtering but back here in the Facade, which makes sense for data manipulation.